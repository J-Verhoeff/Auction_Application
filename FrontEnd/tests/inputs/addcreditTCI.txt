login
admin
addcredit
-1
addcredit
1001
addcredit
100
userNA
addcredit
200
user
addcredit
900
admin
logout
login
admin
addcredit
900
user
logout
login
user
addcredit
100
logout
exit