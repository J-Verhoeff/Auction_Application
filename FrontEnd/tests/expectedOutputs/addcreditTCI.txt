
Welcome to the Auction App!
Start entering transactions
To exit the application type 'exit'

Options:
	login
Transaction: User: 
Welcome admin!
User Type: AA
Available Credit: 100.00$

Options:
	logout
	create
	delete
	advertise
	bid
	refund
	addcredit
Transaction: Amount of credit: Credit amount must be a number between 0.01 and 1000.00!

Options:
	logout
	create
	delete
	advertise
	bid
	refund
	addcredit
Transaction: Amount of credit: Credit amount must be a number between 0.01 and 1000.00!

Options:
	logout
	create
	delete
	advertise
	bid
	refund
	addcredit
Transaction: Amount of credit: Username: User not found!

Options:
	logout
	create
	delete
	advertise
	bid
	refund
	addcredit
Transaction: Amount of credit: Username: Successfully added credit!
Current credit amount: 200.00 for user

Options:
	logout
	create
	delete
	advertise
	bid
	refund
	addcredit
Transaction: Amount of credit: Username: Successfully added credit!
Current credit amount: 1000.00 for admin

Options:
	logout
	create
	delete
	advertise
	bid
	refund
	addcredit
Transaction: 
Welcome to the Auction App!
Start entering transactions
To exit the application type 'exit'

Options:
	login
Transaction: User: 
Welcome admin!
User Type: AA
Available Credit: 100.00$

Options:
	logout
	create
	delete
	advertise
	bid
	refund
	addcredit
Transaction: Amount of credit: Username: Successfully added credit!
Current credit amount: 900.00 for user

Options:
	logout
	create
	delete
	advertise
	bid
	refund
	addcredit
Transaction: 
Welcome to the Auction App!
Start entering transactions
To exit the application type 'exit'

Options:
	login
Transaction: User: 
Welcome user!
User Type: FS
Available Credit: 0.00$

Options:
	logout
	advertise
	bid
	addcredit
Transaction: Amount of credit: Successfully added credit!
Current credit amount: 100.00 for user

Options:
	logout
	advertise
	bid
	addcredit
Transaction: 
Welcome to the Auction App!
Start entering transactions
To exit the application type 'exit'

Options:
	login
Transaction: Goodbye!
